symphony-bdk-core/src/main/java/com/symphony/bdk/core/auth/impl/AuthSessionOboCertImpl.java

```diff
@@ -0,0 +1,75 @@
+package com.symphony.bdk.core.auth.impl;
+
+import com.symphony.bdk.core.auth.AuthSession;
+
+import com.symphony.bdk.core.auth.OboAuthenticator;
+import com.symphony.bdk.core.auth.exception.AuthUnauthorizedException;
+
+import lombok.NonNull;
```

Comment:
do we have that NonNull annotation on purpose (vs the javax one)?
if so maybe we should explicit when we use that one vs the other

Reply:
Import issue I guess, should be `javax.annotation.Nonnull`

Reply:
NonNull from lombok actually helps us validate the configuration which could be useful here
(Although we perhaps want to provider our own validation for more complex use cases and checks or use javax validation)